saleor/core/tasks.py

```diff
@@ -1,8 +1,20 @@
+import datetime
+
 from django.core.files.storage import default_storage
 
 from ..celeryconf import app
+from .models import EventPayload, EventTask
 
 
 @app.task
 def delete_from_storage_task(path):
     default_storage.delete(path)
+
+
+@app.task
+def delete_event_payloads():
+    seven_days_ago = datetime.date.today() - datetime.timedelta(days=7)
```

Comment:
IMO, this should be a const value or variable in the settings.py file.  We should be able to control this time range
You could do something like we did for jwt tokens:
```
settings.py
JWT_TTL_REFRESH = timedelta(seconds=parse(os.environ.get("JWT_TTL_REFRESH", "30 days")))
```